meetupselector/settings/default.py

```diff
@@ -46,7 +46,8 @@
 
 OUR_APPS = [
     "meetupselector.user",
-    "meetupselector.talks",
+    "meetupselector.talks", 
+    "meetupselector.proposals"
```

Comment:
As a good practice, it is recommended to add a trailing comma at the end of multi-line dictionaries, lists, tuples ...
I mean:
```python
OUR_APPS = [
    "meetupselector.user",
    "meetupselector.talks",
    "meetupselector.talks", 
    "meetupselector.proposals",
]
```
Some lints would raise an error when missing trailing comma.

Reply:
Good to know, thanks!